docs/source/user/description.md

```diff
@@ -0,0 +1,14 @@
+# What is PySkinDose?
+
+PySkinDose is an open-source Python™ package for RDSR-based skin dose estimation. This system 
+translates air kerma at the PERP for all irradiation event to estimate entrance skin dose for 
+all irradiated surfaces on computational phantoms. The phantoms are oriented with the X-ray 
+source by geometric parameters found within the RDSR, and conversion from air Kerma at the PERP 
+to skin dose is further supported by correction factors found in literature, as well as a limited 
+number of in-clinic measurements (e.g., validation of the coordinate system, table and pad 
```

Comment:
remove parenthesis and add comma before _e.g._

Reply:
ok